setup/pi/configure.sh

```diff
@@ -251,6 +251,25 @@ function install_matrix_packages () {
   pip3 install matrix-nio
 }
 
+function check_signal_configuration () {
+  if [ "${SIGNAL_ENABLED:-false}" = "true" ]
+  then
+    if [ -z "${SIGNAL_URL+x}" ] || [  "${SIGNALTO_NUM+x}" = "country_code_and_number_configured_with_signal" ] || [  "${SIGNAL_FROM_NUM+x}" = "country_code_and_number_configured_with_signal"  ]
+    then
+      log_progress "STOP: You're trying to setup Signal but didn't provide a URL."
+      log_progress "Define the variables like this:"
+      log_progress "export URL_PORT=put_protocol_ip/hostname_portnumber"
+      log_progress "export TO_NUM=put_phone_number_associated_with_signal_including_country_code"
+      log_progress "export FROM_NUM=put_phone_number_associated_with_signal_including_country_code_to_send_to"
```

Comment:
This is not what these variables are actually called

Reply:
great catch! my bad. Those variables have been updated.